packages/dynamite/dynamite/pubspec.yaml

```diff
@@ -1,5 +1,13 @@
 name: dynamite
-version: 1.0.0
+version: 0.1.0-dev.0
+description: The helper package used at runtime by code generated with the OpenAPI generator dynamite.
+homepage: https://github.com/nextcloud/neon
+repository: https://github.com/nextcloud/neon/tree/main/packages/dynamite/dynamite
+issue_tracker: https://github.com/nextcloud/neon/issues
```

Comment:
The issue tracker is not needed when the repository points to GH (https://dart.dev/tools/pub/pubspec#issue-tracker)